rules-reviewed/technology-usage/embedded-framework-technology-usage.windup.xml

```diff
@@ -1050,17 +1050,17 @@
                 </technology-identified>
             </perform>
         </rule>
-        <rule id="technology-usage-embedded-framework-08950">
+        <rule id="technology-usage-embedded-framework-09100">
```

Comment:
It looks like all the technology related rules are overwriting the Redis one instead of being added.
Like in previous PR, I think this should be fixed.